AEPAssurance/Source/QuickConnect/QuickConnectView.swift

```diff
@@ -317,9 +180,115 @@ public class QuickConnectView {
+            descriptionTextView.rightAnchor.constraint(equalTo: baseView.rightAnchor),
+            descriptionTextView.topAnchor.constraint(equalTo: baseView.topAnchor, constant: uiConstants.HEADER_HEIGHT + uiConstants.DESCRIPTION_TEXTVIEW_TOP_MARGIN),
+            descriptionTextView.heightAnchor.constraint(equalToConstant: uiConstants.DESCRIPTION_TEXTVIEW_HEIGHT)
+        ])
+        
+        baseView.addSubview(connectionImageView)
+        NSLayoutConstraint.activate([
+            connectionImageView.leftAnchor.constraint(equalTo: baseView.leftAnchor),
+            connectionImageView.rightAnchor.constraint(equalTo: baseView.rightAnchor),
+            connectionImageView.topAnchor.constraint(equalTo: descriptionTextView.bottomAnchor, constant: uiConstants.CONNECTION_IMAGE_TOP_MARGIN),
+            connectionImageView.heightAnchor.constraint(equalToConstant: uiConstants.CONNECTION_IMAGE_HEIGHT)
+        ])
+        
+        baseView.addSubview(buttonStackView)
+        NSLayoutConstraint.activate([
+            buttonStackView.leadingAnchor.constraint(equalTo: baseView.leadingAnchor, constant: 40.0),
+            buttonStackView.trailingAnchor.constraint(equalTo: baseView.trailingAnchor, constant: -40.0),
+            buttonStackView.topAnchor.constraint(equalTo: connectionImageView.bottomAnchor, constant: uiConstants.BUTTON_HOLDER_TOP_MARGIN),
+            buttonStackView.heightAnchor.constraint(equalToConstant: uiConstants.BUTTON_HOLDER_HEIGHT)
+        ])
+        
+        buttonStackView.addArrangedSubview(cancelButton)
+        NSLayoutConstraint.activate([
+            cancelButton.heightAnchor.constraint(equalToConstant: uiConstants.CANCEL_BUTTON_HEIGHT)
+        ])
+        
+
+        buttonStackView.addArrangedSubview(connectButton)
+        NSLayoutConstraint.activate([
+            connectButton.heightAnchor.constraint(equalToConstant: uiConstants.CANCEL_BUTTON_HEIGHT)
+        ])
+        initialState()
+        
+        baseView.addSubview(adobeLogo)
+        NSLayoutConstraint.activate([
+            adobeLogo.leftAnchor.constraint(equalTo: baseView.leftAnchor),
+            adobeLogo.rightAnchor.constraint(equalTo: baseView.rightAnchor),
+            adobeLogo.bottomAnchor.constraint(equalTo: baseView.bottomAnchor, constant: uiConstants.ADOBE_LOGO_IMAGE_BOTTOM_MARGIN),
+            adobeLogo.heightAnchor.constraint(equalToConstant: uiConstants.ADOBE_LOGO_IMAGE_HEIGHT)
+        ])
+        
+        self.baseView.frame.origin.y = window.frame.size.height
+        UIView.animate(withDuration: 0.2,
+                       delay: 0,
+                       options: [.curveEaseInOut],
+                       animations: { [] in
+            self.baseView.frame.origin.y = 0
+            self.baseView.backgroundColor = UIColor(red: 47.0/256.0, green: 47.0/256.0, blue: 47.0/256.0, alpha: 1)
+        }, completion: nil)
+            
+    }
+    
+    func sessionConnecting() {
+        // TODO: - No op?
```

Comment:
nit resolve todo

Reply:
This is resolved in: https://github.com/cdhoffmann/aepsdk-assurance-ios/pull/3